packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/attach/telecom-telephony-alias-portabilit-portabilities-attach.controller.js

```diff
@@ -0,0 +1,51 @@
+import endsWith from 'lodash/endsWith';
+
+angular.module('managerApp').controller('TelecomTelephonyServicePortabilityMandateAttachCtrl',
+  class TelecomTelephonyServicePortabilityMandateAttachCtrl {
+    constructor($q, $stateParams, $timeout, $translate, $uibModalInstance, OvhApiTelephony,
+      TucToast, data) {
+      this.$translate = $translate;
+      this.$uibModalInstance = $uibModalInstance;
```

Comment:
Please use a state for your modal

Reply:
no it's a big code rupture for the telecom manager with a big link with angularjs